x/cronos/keeper/evm_log_handlers.go

```diff
@@ -156,3 +177,42 @@ func (h EthereumTransferHandler) Handle(ctx sdk.Context, contract common.Address
 	}
 	return nil
 }
+
+// IbcTransferHandler handles `__CronosSendToIbc` log
+type IbcTransferHandler struct {
+	cronosKeeper Keeper
+}
+
+func NewIbcTransferHandler(cronosKeeper Keeper) *IbcTransferHandler {
+	return &IbcTransferHandler{
+		cronosKeeper: cronosKeeper,
+	}
+}
+
+func (h IbcTransferHandler) EventID() common.Hash {
+	return IbcTransferEvent.ID
+}
+
+func (h IbcTransferHandler) Handle(ctx sdk.Context, contract common.Address, data []byte) error {
+	unpacked, err := IbcTransferEvent.Inputs.Unpack(data)
+	if err != nil {
+		// log and ignore
+		h.cronosKeeper.Logger(ctx).Info("log signature matches but failed to decode")
+		return nil
+	}
+
+	denom, found := h.cronosKeeper.GetDenomByContract(ctx, contract)
+	if !found && !types.IsValidIBCDenom(denom) {
+		return fmt.Errorf("contract %s is not connected to native token", contract)
+	}
+
+	contractAddr := sdk.AccAddress(contract.Bytes())
+	recipient := unpacked[0].(string)
```

Comment:
Validate its a valid cosmos address?

Reply:
yeah but validation is quite limited since we dont know the prefix